boards/nrf51dk/chip_layout.ld

```diff
@@ -3,6 +3,6 @@ ROM_LENGTH  = 128K;
 PROG_ORIGIN = 0x00020000;
 PROG_LENGTH = 128K;
 RAM_ORIGIN  = 0x20000000;
-RAM_LENGTH  = 16K;
```

Comment:
Revert this, because nRF51 comes with 16kB or 32kB RAM and we want to be on the safe side.